src/Symfony/Component/HttpFoundation/StreamedResponse.php

```diff
@@ -26,9 +26,10 @@
  */
 class StreamedResponse extends Response
 {
-    protected $callback;
-    protected $streamed;
-    private bool $headersSent;
+    protected \Closure $callback;
```

Comment:
`setCallback()` accepts any callable. You need to make sure we always assign a closure:

```php
public function setCallback(callable $callback): static
{
    $this->callback = $callback(...);
    return $this;
}
```

In addition to this, in `sendContent()` we check if `$this->callback` is null.

https://github.com/symfony/symfony/blob/f10050724a1d2b5a9d57a30aa730be487d620283/src/Symfony/Component/HttpFoundation/StreamedResponse.php#L90-L92

We might want to change this to an `isset()` check, I suppose.